Hestia/app/src/main/java/com/rugged/application/hestia/Peripheral.java

```diff
@@ -0,0 +1,39 @@
+package com.rugged.application.hestia;
+
+import java.util.UUID;
+
+public class Peripheral {
+    private String type;
+    private UUID uuId;
```

Comment:
Documentation comments would be especially useful in this file, as it is somewhat confusing to have two different types of identifier variables. I suppose you use UUID for your serializing? What is the use of id though?